src/components/application_manager/src/resumption/resumption_data_json.cc

```diff
@@ -148,32 +160,45 @@ void ResumptionDataJson::IncrementIgnOffCount() {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock autolock(resumption_lock_);
   Json::Value to_save = Json::arrayValue;
-  for (Json::Value::iterator it = GetSavedApplications().begin();
-       it != GetSavedApplications().end();
-       ++it) {
-    if ((*it).isMember(strings::ign_off_count)) {
-      Json::Value& ign_off_count = (*it)[strings::ign_off_count];
-      const uint32_t counter_value = ign_off_count.asUInt();
-      ign_off_count = counter_value + 1;
-    } else {
-      LOG4CXX_WARN(logger_, "Unknown key among saved applications");
-      Json::Value& ign_off_count = (*it)[strings::ign_off_count];
-      ign_off_count = 1;
+  Json::Value dictionary;
+  {
+    resumption::LastStateAccessor accessor =
+        last_state_wrapper_->get_accessor();
+    dictionary = accessor.GetData().dictionary();
+    Json::Value& saved_apps = GetSavedApplications(dictionary);
+    for (Json::Value::iterator it = saved_apps.begin(); it != saved_apps.end();
```

Comment:
@LitvinenkoIra rename it to app or json_app

Reply:
@LuxoftAKutsan done 86d7ed7ca314f780ee977f843c0bacff8b471c4a